src/odinapi/views/level2.py

```diff
@@ -534,6 +535,8 @@ def _fetch(self, version, project, freqmode):
         for scan in scans:
             scan['Date'] = time_util.stw2datetime(
                 scan['ScanID']).date().isoformat()
+            scan['StartTime'] = time_util.stw2datetime(
+                scan['ScanID']).time().isoformat()
```

Comment:
would it not be easier for the sorting effort if it was the complete datetime rather than just the time?

Reply:
You are probably correct. But it also a bit odd to have both date and datetime in the response. The reason we do not want to update Date to datetime is that we probably will break someones code. Donal promised to have a look at this and we can see what he says.

Reply:
I think that if we return a datetime it is unlikely to break any code. The date is still there and I am not sure that so many use this info in the endpoint. 

Can we not get the middle timeout looking in the L2 databases.  We have presumably the list of spectra in the scan in the l1 database.

Reply:
we can get the middle time from the L2 databases. The query will be much slower though (I do not really know by how much but it should be significant) and a bit more complicated to implement.

Reply:
I meant to write **without** looking in the level 2 databases.

Reply:
ok, as it is now we can make one query to the L2i database table and get all data we need for the current implementation. It seems like that if we want to get the mean time of the scan we first need to make this query and then make a new query for each scan to either the L1 database or to the l2 database. And as we can get up to 1000 scans in the first query, we may have to make 1000 new queries. This is what I am worried about.

Reply:
Ok don't make it complicated the start time will do in a date time. return

Reply:
So the Scans endpoint uses L2i ? why not use the L2anc?  I am not sure which databases you have nowadays

Reply:
the databases and endpoints are not exactly the same. L2anc is not a database table. L2i is an overview table with one document for each project,freqmode,and scanid. More detailed data are found in L2 tables, but there we have one document for each project, freqmode, product, and altitude or something like this, which makes it a bit complicated to extract overview data from L2 tables. I think it is designed to allow for an altitude filtering of data.

Reply:
ok lets go with the simplest date time is fins as I said